### ISSUE TITLE
confused about input type, why use bgr?

### ISSUE THREAD

**Issue by LCorleone** [auroua/InsightFace_TF#60]
hi, great work!
i am confused why you use the bgr input instead of rgb. It seems that the insightface project uses the rgb input? Why you change the type? any tricks?
thanks!

**Comment by LCorleone** [comment]
no, the dataset of the insightface project (.bin .rec) is using bgr format, but in mx2tfrecords.py you just get the array (here is bgr). but in parse_function, you get r,g,b via tf.split and concat b,g,r .
is this wrong?

**Comment by muyoucun** [comment]
i also find that. dataset of the insightface project is bgr format, and after mx2tfrecods.py the input image is rgb 
but in load_bin, you just use cv2.cvtColor(img, cv2.COLOR_RGB2BGR), so the images in lfw are bgr.

**Comment by erlendd** [comment]
I just extracted some of the image files out of lfw.bin to take a look at them (used mx.imdecode and just wrote them to disk using imageio) the images definitely look like they're RGB formatted to me.
